app/assets/javascripts/thredded.es6

```diff
@@ -1,18 +1,59 @@
+//= require babel/polyfill
 //= require jquery
 //= require jquery_ujs
 //= require_tree ./thredded/vendor
 //= require_tree ./thredded
 
-class Thredded {
-  constructor() {
-    this.topicForm = new ThreddedTopicForm();
-    this.privateTopicForm = new ThreddedPrivateTopicForm();
-    this.postForm = new ThreddedPostForm();
-    this.kbShortcuts = new ThreddedKeyboardShortcuts();
-    this.timestamps = new ThreddedTimeStamps();
-    this.searchBar = new ThreddedSearchBar();
-    this.currentlyOnline = new ThreddedCurrentlyOnline();
+(function() {
```

Comment:
Yes, this is a self-executing anonymous function, so that the global template is not polluted. A better approach is to use [ES6 modules](http://exploringjs.com/es6/ch_modules.html), but I haven't figured out how to make them work with sprockets-es6 yet.